src/test/suite-timeout.ts

```diff
@@ -0,0 +1,23 @@
+import test, { setTestSuiteOptions } from '../index'
```

Comment:
```suggestion
This file can be used as a manual test, because these things are difficult to write automated tests for.

The expected output for this file is...
--
  |  
  | You can run this file to see the behavior if you set test suite options multiple times, and it should throw an error.


import test, { setTestSuiteOptions } from '../index'
```

Reply:
2b17ba1